package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "IonicCordovaRTMPandHLS",
+  "version": "0.0.1",
+  "description": "Ionic Cordova Plugin (RTMP/HLS)",
+  "cordova": {
+    "id": "cordova.plugin.ionicrtmphls",
+    "platforms": [
+      "android",
+      "ios"
+    ]
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/growthengineering/ionic-cordova-rtmp-hls-plugin.git"
+  },
+  "keywords": [
+    "ecosystem:cordova",
+    "cordova-android",
+    "cordova-ios"
+  ],
+  "author": "GrowthEngineering",
```

Comment:
I'm thinking changing the author to **Growth Engineering Technologies** what do you think?

Reply:
Agreed, I have changed